app-ios/tutanota/Sources/AppDelegate.swift

```diff
@@ -5,41 +5,37 @@ class AppDelegate : UIResponder,
                     UIApplicationDelegate,
                     UNUserNotificationCenterDelegate {
   var window: UIWindow?
-  
+
   private var pushTokenCallback: ResponseCallback<String>?
   private let userPreferences = UserPreferenceFacade()
   private var alarmManager: AlarmManager!
   private var viewController: ViewController!
-  
+
   func registerForPushNotifications() async throws -> String {
-    #if targetEnvironment(simulator)
+#if targetEnvironment(simulator)
     return ""
-    #else
+#else
     return try await withCheckedThrowingContinuation { continuation in
       UNUserNotificationCenter.current()
         .requestAuthorization(
           options: [.alert, .badge, .sound]) { granted, error in
-          if error == nil {
-            DispatchQueue.main.async {
-              self.pushTokenCallback = continuation.resume(with:)
-              UIApplication.shared.registerForRemoteNotifications()
+            if error == nil {
+              DispatchQueue.main.async {
+                self.pushTokenCallback = continuation.resume(with:)
+                UIApplication.shared.registerForRemoteNotifications()
+              }
+            } else {
+              continuation.resume(with: .failure(error!))
             }
-          } else {
-            continuation.resume(with: .failure(error!))
           }
-        }
     }
-    #endif
+#endif
   }
-  
-  func application(
-    _ application: UIApplication,
-    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]?
-  ) -> Bool {
-    TUTSLog("Start Tutanota \(String(describing: launchOptions))")
-    
+
+  func main() {
```

Comment:
I'd at least make it private or maybe would rename it because you *can* actually define a `main()` function (see https://docs.swift.org/swift-book/ReferenceManual/Attributes.html#ID626) and it confused me for a good minute